### DISCUSSION TITLE
How to keep long sequences?

### DISCUSSION THREAD

**Discussion by kheyer** [axolotl-ai-cloud/axolotl#1376] [Q&A]
I'm trying to finetune on a dataset with lots of long sequences. I see the phrase "dropping long sequences" printed during preprocessing.

What config parameter controls the drop length?

Is it possible to keep long sequences?

Ideally I'd like long sequences to be chunked into smaller sequences based on some max length parameter. Is there a way to do this?

**Comment by NanoCode012** [comment]
- Increase the `sequence_len`
- Chunk yourself (`type: completion` does this automatically)

That message happens even if no sequences are dropped.
